Classes/Resource/OnlineMedia/Helpers/RtrHelper.php

```diff
@@ -1,25 +1,12 @@
 <?php
-namespace Saccas\Srgssr\Resource\OnlineMedia\Helpers;
 
-/***
- *
- * This file is part of the "Srgssr" Extension for TYPO3 CMS.
- *
- * For the full copyright and license information, please read the
- * LICENSE.txt file that was distributed with this source code.
- *
- *  (c) 2019 https://www.sac-cas.ch
- *
- ***/
+namespace Saccas\Srgssr\Resource\OnlineMedia\Helpers;
 
 class RtrHelper extends AbstractSrgssrHelper
 {
-    /** @var string */
-    protected $channelName = 'rtr';
+    protected string $channelName = 'rtr';
 
-    /** @var string  */
-    protected $siteHostname = 'www.rtr.ch';
+    protected string $siteHostname = 'www.rtr.ch';
 
-    /** @var string */
     protected $extension = 'rtr';
```

Comment:
see above. But if phpstan doesn't complain here, it's ok for me without type declaration.
```suggestion
    protected string $extension = 'rtr';
```